test/Governance.js

```diff
@@ -359,30 +356,27 @@ contract("Governance", async (accounts) => {
 
         let event = res.logs[0].args;
 
-        await gov.test();
-        await wait(3000);
-        await gov.test();
+        await wait(18000);
```

Comment:
just wanted to know about the logic of having large wait times . is it issue due to the testing on the local blockchain or we will face same issue even if we are testing on the mainnet . if later is the case then we will have to put a modifier for that. what do you think @Edoumou  and @drikssy